TripPlanner - Project/TripPlannerFrontend/src/components/Branding/Logo.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function Logo() {
```

Comment:
Could you change all

```
export default function MyFunction() { ... }
```

to

```
export const MyFunction = () => { ... }
```

From my experience named exports work much better than default ones. VSCode can import them as autosuggestions as you type.

Change from functions to anonymous functions (arrow functions) is also a big yes. Why? You're not using `this` and for the most part, you shouldn't use `this` - it leads to confusion. It's a rule of thumb: if you're not using `this` in your functions, use anonymous function (arrow function)